spec/cardVerification_spec.rb

```diff
@@ -0,0 +1,40 @@
+require './CardValidator'
+describe Card, "#valid" do
+    it "Checks that valid Cards are marked as Valid" do
+        trueValues = ["60141016700078611", "6014111100033006", "6014709045001234", "6014352700000140", "6014355526000020"]
+
+        trueValues.each do |cardNumber|
+            card = Card.new(cardNumber)
+            card.validate
+            expect(card.valid).to eq true
+        end
+    end
+    it "Checks that invalid Cards are marked Invalid" do
+        falseValues = ["6014152705006141", "6014 3555 2900 0028", "6013111111111111", "60143527daves140"]
+
+        falseValues.each do |cardNumber|
+            card = Card.new(cardNumber)
+            card.validate
+            expect(card.valid).to eq false
+        end
+    end
+end
```

Comment:
Three main things that catch my eye here are:

1. The spec structure (can leave this for now but one thing to think about)

To provide more context for the specs, we can use the context method (in terms of functionality it works similar to describe in that you can't call any equality matchers in this space but helps to provide clear documentation. In terms of formatting we can do this:
```
describe Card do
# We only hold test one class per spec file so this block with the class name can just be declared once.
  describe "#valid" do
     context "when provided with input that passes the checksum" do
        it "returns true" do
          ....
       end
     end

     context "when provided with input that fails the checksum" do
       it "returns false" do
         ...

     end
     end
  end
end
```
`context` is generally more helpful when grouping a bunch of similar logic together however, as we can define reusable variables at this level.


2. These tests may be better declared as testing `#validate` rather than the readable ivar `@valid`. This is because validate is the actual functionality here where is `valid` is just the result. This won't require logical changes to your tests as that part is fine - just changing `#valid` to `#validate` in this descriptions.

3. When using `#` followed by a name in Ruby e.g. `#validate`, this is used to symbolize a method. `valid` however is an ivar `@valid`. Since we are changing the tests to `#validate` anyway, this won't matter, but will be helpful to know for the purpose of communication down the track.